tests/Moq.Tests.VisualBasic/IssueReports.vb

```diff
@@ -75,4 +75,40 @@ Public Class IssueReports
 
 	End Class
 
+	Public Class Issue1129
+
+		<Fact>
+		Public Sub Test()
+			Dim failClassMock = New Mock(Of IndexerInterface)()
+
+			failClassMock.SetupAllProperties()
+
+			Assert.False(failClassMock.Object.Value)
+		End Sub
+		Interface IndexerInterface
+			ReadOnly Property SystemDefault() As Boolean
+			Property Value() As Boolean
+			Property Value(ByVal OverrideLevel As Integer) As Boolean
+			Property Value(ByVal OverrideLevel As Integer, ByVal OverrideID As String) As Boolean
+		End Interface
+	End Class
+
+	Public Class Issue1129
+
+		<Fact>
+		Public Sub Test()
+			Dim failClassMock = New Mock(Of IndexerInterface)()
+
+			failClassMock.SetupAllProperties()
+
+			Assert.False(failClassMock.Object.Value)
+		End Sub
+		Interface IndexerInterface
+			ReadOnly Property SystemDefault() As Boolean
+			Property Value() As Boolean
+			Property Value(ByVal OverrideLevel As Integer) As Boolean
+			Property Value(ByVal OverrideLevel As Integer, ByVal OverrideID As String) As Boolean
+		End Interface
+	End Class
+
```

Comment:
This class is duplicated which causes compilation to fail. Please delete the duplicate.